Sources/Core/Yoakke.Lexer/Regexes.cs

```diff
@@ -42,7 +42,7 @@ public static class Regexes
         /// Note that this CANNOT represent all IEEE-754 floating point values, as special values such as infinity or NaN are not supported.
         /// </remarks>
         /// <seealso cref="IeeeFloatLiteral"/>
-        public const string RealNumberLiteral = "(([0-9]*.)|([0-9]+[0-9_]*.))?[0-9]+[0-9_]*((e|E)(+|-)?[0-9]+)?";
+        public const string RealNumberLiteral = "(([0-9]*.)|([0-9]+[0-9_]*.))?[0-9]+[0-9_]*((e|E)(\\+|-)?[0-9]+)?";
```

Comment:
I believe when escapes come in, it's useful to make the string a verbatim string to avoid surprises.